input/resources/au-core-client.xml

```diff
@@ -494,6 +494,9 @@
 <li>
 <strong>SHALL</strong> support <code>read</code>,<code>search-type</code>.
 </li>
```

Comment:
This change only does the narrative. Also need to add to the underlying resources. MAY that is in the narrative needs to be on each resource with a conformance obligation extension of MAY.